apps/sms/test/marionette/conversation_test.js

```diff
@@ -314,4 +312,98 @@ marionette('Conversation Panel Tests', function() {
       assert.equal(downloadedMessage.content.trim(), 'Attachment');
     });
   });
+
+  suite('Edit Mode', function() {
+    var thread, conversationView;
+    setup(function() {
+      thread = ThreadGenerator.generate({
+        numberOfMessages: 4
+      });
+      messagesApp.launch();
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
+      storage.setContactsStorage();
+      var inboxView = new InboxView(client);
+      conversationView = inboxView.goToConversation(thread.id);
+
+    });
+    
+    test('Enter and leave edit', function () {
+
+      conversationView.enterEditMode();
+      conversationView.messages.forEach(function(message) {
+        assert.isTrue(message.isInEditMode);
+      });
+
+      conversationView.exitEditMode();
+      conversationView.messages.forEach(function(message) {
+        assert.isFalse(message.isInEditMode);
+      });
+
+    });
+
+    test.only('select/deselect all', function () {
```

Comment:
@azasypkin The header which has the select all button disappears on selecting a message :cry: